src/script/entity/Conversation.ts

```diff
@@ -454,7 +464,7 @@ export class Conversation {
           return truncate(joinedNames, maxLength, false);
         }
 
-        const hasUserIds = !!this.participating_user_ids().length;
+        const hasUserIds = this.participating_user_ids().length + this.participatingQualifiedUserIds().length > 0;
```

Comment:
How about exposing this value (`this.participating_user_ids().length + this.participatingQualifiedUserIds().length`) in a getter for `Conversation.ts`?